src/Microsoft.PowerShell.Commands.Management/commands/management/Clipboard.cs

```diff
@@ -59,102 +75,68 @@ private static string StartProcess(
             return stdout;
         }
 
-        public static string GetText()
+        public static string GetText(Hashtable clipboardActions)
         {
-            if (_clipboardSupported == false)
-            {
-                return _internalClipboard ?? string.Empty;
-            }
+            // need errors for null clipboardActions
+            // need errors for null clipboardActions["Paste"]
```

Comment:
Need to raise exceptions for a null `clipboardActions` (`$PSClipboardActions`) and null `clipboardActions["Paste"]`, currently shows a somewhat cryptic `Object reference not set to an instance of an object.` (NRE), correct?

Also need guidence on language for exception text (see line 89)